test/unittests/mongoBackend/mongoUpdateContext_withOnchangeSubscriptions_test.cpp

```diff
@@ -289,8 +289,88 @@ static void prepareDatabase(void) {
   connection->insert(SUBSCRIBECONTEXT_COLL, sub2);
   connection->insert(SUBSCRIBECONTEXT_COLL, sub3);
 
+  /* Given that preparation including csubs, we have to init cache */
+  subscriptionCacheInit(DBPREFIX);
```

Comment:
You mean that we could first fill mongo with patterned-onchange subscriptions and after that fill the cache from the db?
Yes, we can do that. The method already exists.

About the two calls to subscriptionCacheInit (yours in mongoUpdateContext_withOnchangeSubscriptions_test.cpp and mine in main_UnitTest.cpp) it got like that after a merge, he had added the call both of us ...
subscriptionCacheInit must be called, of course.
Is it not better to call subscriptionCacheInit only in one place, from main_UnitTest.cpp?